### ISSUE TITLE
[Bookworm] fwutil show status is showing extra output 

### ISSUE THREAD

**Issue by vivekrnv** [sonic-net/sonic-buildimage#17063]
#### Steps to reproduce the issue:

Run `sudo fwutil show status`

#### Describe the results you received:

```
root@r-bulldog-03:/home/admin# sudo fwutil show status
/usr/local/lib/python3.11/dist-packages/fwutil/lib.py:902: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if boot is "none" or boot in data:
mount: (hint) your fstab has been modified, but systemd still uses
       the old version; use 'systemctl daemon-reload' to reload.
Chassis    Module    Component    Version                  Description
---------  --------  -----------  -----------------------  ----------------------------------------
MSN2100    N/A       ONIE         2022.08-5.3.0010-9600    ONIE - Open Network Install Environment
                     SSD          S19903Mi                 SSD - Solid-State Drive
                     BIOS         0ACIF007_02.02.002_9600  BIOS - Basic Input/Output System
                     CPLD1        CPLD000087_REV0600       CPLD - Complex Programmable Logic Device
                     CPLD2        CPLD000075_REV0600       CPLD - Complex Programmable Logic Device
```

#### Describe the results you expected:

No additional output

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by saiarcot895** [comment]
The syntax warning error has been fixed in sonic-net/sonic-utilities#3013.

The mount warning/hint appears to be because the modified timestamp of the `/etc/fstab` file is newer than when systemd was last reloaded. This might be because there is a time sync issue between the image build server/environment and the time on the device.

**Comment by gechiang** [comment]
@vivekrnv issue is fixed already. Please confirm and close if satisfied
